dev-db/mariadb-connector-odbc/mariadb-connector-odbc-3.1.6.ebuild

```diff
@@ -0,0 +1,57 @@
+# Copyright 2018-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake-utils multilib-minimal
+
+DESCRIPTION="MariaDB Connector/ODBC"
+HOMEPAGE="https://downloads.mariadb.org/connector-odbc/"
+SRC_URI="https://downloads.mariadb.org/interstitial/connector-odbc-${PV}/${P}-ga-src.tar.gz"
+
+LICENSE="LGPL-2.1"
+SLOT="0/3.0"
+KEYWORDS="~amd64 ~x86"
+IUSE="ssl gnutls"
+
+S="${S}-ga-src"
+
+DEPEND="=dev-db/mariadb-connector-c-$(ver_cut 1-2)*"
+RDEPEND="${DEPEND}"
+BDEPEND=""
+
+multilib_src_configure() {
+	append-cppflags $(mariadb_config --cflags)
+	local mycmakeargs=(
+	-DWITH_SSL:STRING=$(usex ssl $(usex gnutls GNUTLS OPENSSL) OFF)
+	-DMARIADB_LINK_DYNAMIC=YES
+	-DUSE_SYSTEM_INSTALLED_LIB=YES
+	#-DCMAKE_C_FLAGS="$(mariadb_config --cflags)"
+	)
+	cmake-utils_src_configure
+}
+
+multilib_src_compile() {
+	cmake-utils_src_compile
+}
+
+multilib_src_install() {
+	cmake-utils_src_install
+}
+
+multilib_src_install_all() {
+	dodir /usr/share/${PN}
+	sed -e "s,lib/lib,$(get_libdir)/lib,g" < "${FILESDIR}/odbcinst.ini" > "${D}/usr/share/${PN}/odbcinst.ini"
```

Comment:
I really don't like relying on sed to do this. Can't you make the modification in src_prepare, then use proper do* helper functions here? Allows you to skip dodir too.

Reply:
I agree.  However ... multilib.  So if this can be sensibly done I'd love for someone to show me (or point me).

Reply:
```
/tmp/portage/dev-db/mariadb-connector-odbc-3.1.6/temp/environment: line 2324: /tmp/portage/dev-db/mariadb-connector-odbc-3.1.6/files/odbcinst.ini: No such file or directory
```

Wouldn't simple 
```
src_prepare() {                                                                       
    default                                                                           
                                                                                      
    cp "${FILESDIR}/odbcinst.ini" . || die                                            
    sed -e "s,lib/lib,$(get_libdir)/lib,g" odbcinst.ini || die                        
}                                                                                     
                                                                                      
...                                                                                   
                                                                                      
multilib_src_install_all() {                                                          
    insinto /usr/share/${PN}                                                          
    doins odbcinst.ini                                                                
}
```
work? Why this even is multilibbed, is there a 32bit binary requiring this dep?

Reply:
This was based off of the myodbc and mariadb-connector-c packages, both of which is multilib.  I don't personally have a specific need for multilib.

Actually committed the odbcinst.ini file which I missed in the PR.

Had to modify your way slightly, but it works now.